src/components/styles/sidenav.module.css

```diff
@@ -0,0 +1,75 @@
+.navigation{
+  position: sticky;
+  top: 6rem;
+  width: 75px;
```

Comment:
here i think you can use rem. if you don't use here. what was reason.

Reply:
Thanks fro pointing it out, I have fixed it in the latest commit